src/ReflectionClosure.php

```diff
@@ -626,6 +626,10 @@ protected function getHashedFileName()
      */
     protected function getFileTokens()
     {
+        if (stripos($this->getFileName(), "eval()'d code") > 0) {
+            // In this use case the code is not in a file, it is loaded from the database and eval()'d, so no filename, no tokens.
+            return NULL;
```

Comment:
`null` should be lowercase. ;)

Reply:
ok I will update that, every project has it's own standard.  I will push that change up right away.